src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,71 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book() {
+
+    }
+
+    public Book(String title, BigDecimal price) {
+        this.title = title;
+        this.price = price;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
equals, hashCode and toString usually are placed after get and set methods 

Why do you need equals and hashCode here?